### DISCUSSION TITLE
Problems downloading data from http://scanorama.csail.mit.edu/data.tar.gz

### DISCUSSION THREAD

**Discussion by AnneHartebrodt** [brianhie/scanorama#94] [Q&A]
Dear Brian,

I am having troubles downloading the data from http://scanorama.csail.mit.edu/data.tar.gz
I used this command, but I have the suspicion that the link is broken:

```
wget http://scanorama.csail.mit.edu/data.tar.gz
tar xvf data.tar.gz

```
I also tried via the web interface.
 
Could you give me a pointer as to how to correctly download the data?

Best,
Anne

**Comment by brianhie** [comment]
Weird, I think there's some problem with the URL lookup -- please try downloading from http://cb.csail.mit.edu/cb/scanorama/data.tar.gz instead, that seems to still work! I'll also update the README.

**Reply by AnneHartebrodt** [reply]
Great that worked! Thanks!
